docs/controls/access_control/ac-2(7).md

```diff
@@ -0,0 +1,33 @@
+# Access Control
+## AC-02(07) - Account Management | Role-Based Schemes
+
+The organization:  
+a. Establishes and administers privileged user accounts in accordance with a 
+role-based access scheme that organizes allowed information system access and 
+privileges into roles;  
+b. Monitors privileged role assignments; and  
+c. Takes [Assignment: organization-defined actions] when privileged role 
+assignments are no longer appropriate.
+
+### TDP Implementation
+
+a. [Pull Request #95](https://github.com/HHS/TANF-app/pull/95) establishes the
+implementation of roles and permissions in the TDP application. It provides 
+endpoints for the creating, editing and reading existing roles and permissions.
```

Comment:
There is only one endpoint, and it is for reading existing roles and permissions.

Reply:
Updated per the comment.